packages/salesforcedx-vscode-core/test/vscode-integration/orgBrowser/metadataOutlineProvider.test.ts

```diff
@@ -416,6 +529,32 @@ function compareNodes(actual: BrowserNode[], expected: any[]) {
   });
 }
 
+function getExpected(label1: string, label2: string, type: NodeType) {
+  return [
+    {
+      label: label1,
+      fullName: label1,
+      type
+    },
+    {
+      label: label2,
+      fullName: label2,
+      type
+    }
+  ];
+}
+
+function getMetadataObject(label: string) {
```

Comment:
Refactor looks good - thanks 👍 . As the metadata object returned from here seems to be used only creating a new BrowserNode, wonder if it would make sense to extract creation of BrowserNode as the util method here instead.
But I will leave it up to you since I failed to notice this in the last review iteration.

Reply:
Thank you! Apart from metadataObject, other parameters like label, nodeType or fullName are all different each time we are creating a BrowserNode. `BrowserNode()` itself is taking care of the node creation. I think it's better this way.